File: t-bank/real-tasks/task-1.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
async function solve() {
    const input = await readLine();
    // 1 <= n, m <= 31
    const [n, m] = input.split(" ").map(Number)
    if (n <= 7) {
        console.log(m + (n + Math.abs(n - 7)))
    } else {
        console.log(n - 7)
    }

    rl.close()
}
function readLine() {
    return new Promise((res) => {
        rl.once("line", res);
    });
}

solve();

File: t-bank/test-tasks/task-1.py
# A - Payment for subscription


# a = int(input())
# b = int(input())
# c = int(input())
# d = int(input())

a, b, c, d = map(int, input().split(" "))

total = a
traffic = b - d

if traffic < 0:
    total += abs(traffic) * c

print(total)
